### PR TITLE
Adding System.Data.Odbc package and including in metapackage

### PR DESCRIPTION
**Pull request by maryamariyan**

### CODE REVIEW

File: pkg/descriptions.json [hunk 1]
```diff
@@ -455,6 +455,19 @@
     ]
   },
   {
+    "Name": "System.Data.Odbc",
+    "Description": "Provides a collection of classes used to access an ODBC data source in the managed space",
```
Comment (danmosemsft) [inline pkg/descriptions.json]: I'm not sure what `in the managed space` means but I'd probably write `from managed code` ...

File: src/System.Data.Odbc/src/Configurations.props [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netfx-Windows_NT;
+      netstandard;
```
Comment (saurabh500) [inline src/System.Data.Odbc/src/Configurations.props]: Why do we need netstandard here? Wouldn't the rest of the configurations cover supported netstandard on different OS?

Comment (maryamariyan) [reply src/System.Data.Odbc/src/Configurations.props]: The problem we hit when we don't validate against netstandard (and instead validating against concrete frameworks like netfx and netcoreapp, etc.) is that then we don't have a "rid-less" build configuration for the src project.

For the "Rid-less" configuration we are now producing a turd assembly that just throws a PlatformNotSupportedException

Comment (mellinoe) [reply src/System.Data.Odbc/src/Configurations.props]: Yep, that explanation is correct. Mono, for example, supports .NET Standard 2.0 but will not select the RID split packages.

Comment (saurabh500) [reply src/System.Data.Odbc/src/Configurations.props]: Thanks

**Comment by saurabh500** [comment]
Cc @corivera @divega fyi

File: pkg/descriptions.json (hunk 1 above)
Comment (mellinoe) [reply pkg/descriptions.json]: This is more or less what the MSDN page says about the namespace: https://msdn.microsoft.com/en-us/library/system.data.odbc(v=vs.110).aspx

Comment (danmosemsft) [reply pkg/descriptions.json]: ha, fine

**Comment by danmosemsft** [comment]
@weshaggard seem OK?

**Comment by mellinoe** [comment]
Based on the authoring I did for System.Drawing.Common, it all looks good to me. We can always clean up the description in the future if needed. Merging now. Thanks for getting this in, @maryamariyan !

**Comment by divega** [comment]
Hope you saw this @tinchou!

**Comment by tinchou** [comment]
OMG 😍.

Maybe we can try to find that ODBC blog post draft I wrote while I was there? I'd be thrilled to publish it!

**Comment by danmosemsft** [comment]
Maybe when it goes on nuget, not just myget

**Pull request merged**